pyaedt/hfss3dlayout.py

```diff
@@ -158,14 +158,14 @@ def __exit__(self, ex_type, ex_value, ex_traceback):
 
     @aedt_exception_handler
     def create_edge_port(self, primivitivename, edgenumber, iscircuit=True):
-        """Create a new edge port.
+        """Create an edge port.
 
         Parameters
         ----------
         primivitivename : str
-            Name of the primitive.
+            Name of the edge port.
```

Comment:
```suggestion
            Name of the primitive to create the edge port on.
```
@maxcapodi78 for CreateEdgePort, is the primitive name parameter the name of the primitive the edge port is created on?

Reply:
Correct